src/utils.js

```diff
@@ -88,7 +102,9 @@ async function generateOAuthSTSAuthToken (params = {}) {
     technical_account_email: techAccEmail,
     technical_account_id: techAccId,
     ims_org_id: imsOrgId,
-    scopes: scopes.split(',')
+    scopes: [
```

Comment:
Why so tricky? Can't this just be `scopes: finalScopes` ?
or `finalScopes.slice()` ?

Reply:
Originally I just wrote `scopes: finalScopes`, but then I saw below in the auth step it was written this way and decided to keep things consistent. Should work either way, but maybe there's a hidden benefit to using the spread operator? Any idea @shazron? I think this was written when you found the double array bug

Reply:
Going to keep this as is for now, IIWIT